mojaloop/values.yaml

```diff
@@ -313,7 +321,9 @@ central:
                         "debug": "all",
                         "group.id": "central-ledger-kafka",
                         "metadata.broker.list": "$kafka_host:$kafka_port",
-                        "enable.auto.commit": false
+                        "enable.auto.commit": false,
+                        "socket.keepalive.enable": true,
+                        "socket.max.fails": 0
```

Comment:
I would suggest also adding the same kafka configs to the "central" & "centralledger" charts:
- "socket.keepalive.enable": true
- "socket.max.fails": 0